docs/conf.py

```diff
@@ -171,7 +171,7 @@ class CqlLexer(BashLexer):
 BRANCHES = []
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Whitelist pattern for remotes (set to None to use local branches only)
-smv_remote_whitelist = r"^none$"
+smv_remote_whitelist = r"^origin$"
```

Comment:
It's not critical, but we usually don't put in the same series a patch that introduces an error and then fixes it in a following patch. You can merge the two patches (there are several patches like this in this series that can be squashed).
Again, I think this is not critical and would accept such a series, but some other maintainers may be less forgiving ;-)

Reply:
@nyh - please tell me what needs to be done

Reply:
You can use "git rebase -i" to "squash" together patches that shouldn't remain separate - e.g., one big patch which creates a file and then a patch that fixes an error in the first patch can be "squashed" into one patch.
As I said, I don't consider this critical, but this is the normal way we work in scylla.git so some other maintainers might insist on it.